### PR TITLE
add_0.5.1_releasenote and update_what'new/timeline and add new doc_up…

### PR DESCRIPTION
**Pull request by lacrimosaprinz**
…date_mo

## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [x] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #

## What this PR does / why we need it:

### CODE REVIEW

File: docs/cn/MatrixOne/Get-Started/update-standalone-matrixone.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+# **升级单机版 MatrixOne**
+
+MatrixOne 可以通过以下三种方法进行升级：
```
Comment (dengn) [inline docs/cn/MatrixOne/Get-Started/update-standalone-matrixone.md:3]: Add description about this is only about in-0.5 version update. Not for 0.4/0.3/0.2/0.1 -> 0.5 version update.

**Review by dengn — CHANGES_REQUESTED** [review]

File: docs/en/MatrixOne/Get-Started/update-standalone-matrixone.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
+# **Update Standalone MatrixOne**
+
+You can update the standalone MatrixOne version by 3 methods:
+
+- <p><a href="#code_source_update">Method 1: Updating from source</a></p>
+- <p><a href="#binary_packages_update">Method 2: Using binary package to update</a></p>
+- <p><a href="#use_docker_update">Method 3: Using Docker to update</a></p>
+
```
Comment (dengn) [inline docs/en/MatrixOne/Get-Started/update-standalone-matrixone.md:8]: same for english version

**Review by dengn — CHANGES_REQUESTED** [review]

**Review by dengn — APPROVED** [review]

**Comment by guguducken** [comment]
@mergifyio refresh

**Comment by fengttt** [comment]
> @guguducken is not allowed to run commands

@mergifyio refresh

**Comment by fengttt** [comment]
@mergifyio refresh

**Comment by fengttt** [comment]
@mergifyio refresh

**Pull request merged**